libshaderc_spvc/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 project(libshaderc)
 
+set(SPIRV_TOOLS "SPIRV-Tools")
```

Comment:
nit: Why is this needed? I am not saying it isn't needed, I am just not sure what it does.

Reply:
It's not really needed, I was just following what shaderc/third_party/spirv-tools/CMakeLists.txt does.

Reply:
That is fine then.